packages/kyt-core/src/config/webpack.dev.server.js

```diff
@@ -32,10 +58,16 @@ module.exports = options => ({
   },
 
   plugins: [
-    new webpack.BannerPlugin({
-      banner: 'require("source-map-support").install();',
-      raw: true,
-      entryOnly: true,
+    // Prevent creating multiple chunks for the server
+    new webpack.optimize.LimitChunkCountPlugin({
+      maxChunks: 1,
+    }),
+    new webpack.HotModuleReplacementPlugin(),
+    new StartServerPlugin({
+      name: 'main.js',
+      nodeArgs,
```

Comment:
Where was this before? Are we sure we wanna introduce a `main.js` requirement?

Reply:
previously, this was an extra Express server in code - all of that logic has been removed, simplifies the `dev` action considerably. This is what Razzle does: https://github.com/jaredpalmer/razzle/blob/master/packages/razzle/config/createConfig.js#L337

Reply:
the dev entrypoint *is* `main.js`